public/data/Ugoo-Eze.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "UGOO EZE",
+  "type": "personal",
+  "bio": "FULLSTACK WEB DEVELOPER.THANKS TO JAVASCRIPT!",
+  "avatar": "https://github.com/Ugoo-Eze.png",
+  "links": [
+    {
+      "name": "Check my Projects out on Github",
+      "url": "https://github.com/Ugoo-Eze",
+      "icon": "github"
+    },
+    {
+      "name": "Check out my journey on Twitter",
+      "url": "https://twitter.com/DerHerr66739164",
+      "icon": "twitter"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started my Tech Journey",
+      "date": "August 2020",
+      "icon": "Laptop",
```

Comment:
```suggestion
```
For milestones, there's no need of any icon, and also as @Panquesito7 stated there's no laptop icon 😅

Reply:
Actually a icon is prefered/